sdk/keyvault/azure-security-keyvault/tests/recordings/test_examples_secrets.test_example_secrets_backup_restore.yaml

```diff
@@ -0,0 +1,253 @@
+interactions:
+- request:
+    body: client_id=fd341195-73b0-4ba7-bf05-a33d6464f84c&client_secret=Bx7%2FR.J%3FvGhcA%3AE3pvBCAXRGje06%3A%3A%2Ac&grant_type=client_credentials&scope=https%3A%2F%2Fvault.azure.net%2F.default
```

Comment:
Please delete this secret and/or principal, and remove the login request from the recording.